src/Plugin/Field/FieldFormatter/CoolPreview.php

```diff
@@ -40,14 +43,24 @@ public function settingsSummary() {
      * {@inheritdoc}
      */
     public function viewElements(FieldItemListInterface $items, $langcode) {
-        $element = [];
+        $elements = [];
+        $media = $items->getEntity();
+        if (!$media instanceof MediaInterface) {
+            // The formatter is being used on the wrong entity type.
+            // @todo Log this.
```

Comment:
There will need to be an issue about error handling, then we can clean this up.
To me, the `@todo` indicates that the code as it is now is not intentionally this way but because we did not have the capacity or scope to do it properly.
But I can drop it.

Reply:
New issue:
- #32